sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/PrivateService.java

```diff
@@ -163,33 +148,6 @@ public static TPrivate createPrivate(TCompasBay compasBay) {
         return createPrivate(objectFactory.createBay(compasBay));
     }
 
-    /**
-     * Create Private of given type as parameter
-     * @param compasCriteria type of Private to create
-     * @return created Private
-     */
-    public static TPrivate createPrivate(TCompasCriteria compasCriteria) {
-        return createPrivate(objectFactory.createCriteria(compasCriteria));
-    }
-
-    /**
-     * Create Private of given type as parameter
-     * @param compasFlow type of Private to create
-     * @return created Private
-     */
-    public static TPrivate createPrivate(TCompasFlow compasFlow) {
-        return createPrivate(objectFactory.createFlow(compasFlow));
-    }
-
-    /**
-     * Create Private of given type as parameter
-     * @param compasFunction type of Private to create
-     * @return created Private
-     */
-    public static TPrivate createPrivate(TCompasFunction compasFunction) {
```

Comment:
Do NOT remove this method, I need it in RCONF (RSR-520) I am workng on.

Reply:
ok